packages/backend/src/util/parsePublishableKey.ts

```diff
@@ -0,0 +1,56 @@
+import { PublishableKey } from '@clerk/types';
```

Comment:
This file is just a copy of: 
https://github.com/clerkinc/javascript/blob/bfb1825e78cdf9970425dc79f3f9610e8a2699f7/packages/shared/src/utils/keys.ts#L1-L0